packages/core/src/sync-store/postgres/store.ts

```diff
@@ -961,6 +961,7 @@ export class PostgresSyncStore implements SyncStore {
           "blocks.number as block_number",
           "logs.logIndex as log_logIndex",
         ])
+        .where((eb) => this.buildCheckpointCmprs(eb, ">", fromCheckpoint))
```

Comment:
update: i still need to dig into why adding this lower bound breaks this query. by break i mean it makes it return zero rows. 

the corresponding one in sqlite works as expected

Reply:
by works as expected i mean it returns a single row containing the last element in the filter/checkpoint combinations result set